requirements.txt

```diff
@@ -0,0 +1 @@
+mezzanine >= 4.2.0
```

Comment:
This file isn't needed here. It exists in the project templates for Cartridge and Mezzanine, but in those projects it's auto-generated. Adding it to Drum's project template manually would likely mean it doesn't get updated, so let's omit it entirely.